cypress/integration-qe/e2e-createNewCluster.spec.js

```diff
@@ -0,0 +1,57 @@
+describe('Flow', () => {
+  it('start from the /clusters page', () => {
+    // Set CYPRESS_BASE_URL environemnt variable
+    // Example: export CYPRESS_BASE_URL=http://localhost:3000
+    cy.visit('');
+  });
+
+  it('create a cluster named $CLUSTER_NAME', () => {
```

Comment:
Very similar test already exist - `createDummyCluster()` in `shared.js`. Needs to be u[dated for the recently changed UI (pull-secret has been moved). I prefer to reuse&improve such parts as much as possible.

Reply:
The problem with createDummyCluster is that it opens the dialog, and clicks cancel... Then it creates a cluster named "test-infra-cluster" to see that it fails because it expects a cluster with that name to already exist (but it won't fail in our environments)... We need to think together how to share code. I think createDummyCluster should not be in shared.js because it's a very specific test case.

Reply:
We'll move createDummyCluster out of shared.js and make a more general createCluster function

Reply:
For now we'll leave createDummyCluster in shared.js.